code/game/objects/items/robobag.dm

```diff
@@ -32,8 +32,15 @@
 		. += "<span class='notice'>[src] has a [corptag] attached to it.</span>"
 
 /obj/structure/closet/body_bag/cryobag/robobag/update_icon()
+	if(opened)
+		icon_state = "open"
+	else if (contents.len)
+		icon_state = "closed_occupied"
+	else
+		icon_state = "closed_unlocked"
```

Comment:
```suggestion
		icon_state = "closed"
```
In addition, the state this aims at in the DMI does not have a name and needs correcting

Reply:
the unnamed icon state + closed_unlocked (presumably a carryover from lockers in general when lockers were redone a few years ago) is the same for all types of bodybag, so it's consistent(ly weird and fucky) right now so I'll correct this for all of them in that case.

Reply:
freaky, but for another PR